docs/domains/Domain.md

```diff
@@ -193,6 +196,21 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | `annotations` | Map | The annotations to be attached to generated resources. |
 | `labels` | Map | The labels to be attached to generated resources. The label names must not start with 'weblogic.'. |
 
+### Model
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `configMap` | string | WDT config map name. Required. |
```

Comment:
This does not need to be required, even if FromModel.

Change wording to something like "Optional WDT config map, only used if domainHomeSourceType is FromModel."

Reply:
Also, please change corresponding java so that this field is no longer required.